.github/workflows/docs.yaml

```diff
@@ -0,0 +1,46 @@
+name: CI Docs
+
+on:
+  push: # Run on pushes to the default branch
+    branches: [main, docs_job]
+  pull_request_target: # Also run on pull requests originated from forks
+    branches: [main, docs_job]
+  release:
+    types: ['published']
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+
+  Deploy-Pages:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.ref }}
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          architecture: "x64"
+
+      - name: Install packages and dependencies
+        run: pip install hatch
+
+      - name: Deploy Docs
+        run: |
+          # git config user.name "Pankaj Singh"
+          # git config user.email "pankaj.singh@astronomer.io"
```

Comment:
@pankajastro would it be an alternative to set the following Github Action environment variable?
```
        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Use GitHub token for authentication
```

Reply:
I have changed it to dummy user and look like working

Reply:
Would it be worth to check if this would work as well?

Reply:
Testing looks like user.name and email require